src/rgw/rgw_sal_filter.h

```diff
@@ -610,9 +610,9 @@ class FilterZone : public Zone {
     virtual const std::string& get_realm_id() = 0;
 };
 
-class FilterLuaScriptManager : public LuaScriptManager {
+class FilterLuaManager : public LuaManager {
 public:
-  virtual ~FilterLuaScriptManager() = default;
+  virtual ~FilterLuaManager() = default;
 
   virtual int get(const DoutPrefixProvider* dpp, optional_yield y, const std::string& key, std::string& script) = 0;
```

Comment:
shouldn't these be:
`get_script()`, `put_script()` and `del_script()` ?

Reply:
Yeah.  Filter's not being built yet, so it won't get caught.

Reply:
I thought I marked this as resolved.